### ISSUE TITLE
share assets

### ISSUE THREAD

**Issue by robalb** [ananagame/vimsence#7]
could you share the assets you used for the app? i would like to contribute to their design

**Comment by ananagame** [comment]
I can’t, they are in the discord app

**Comment by robalb** [comment]
yeah, i was hoping you could just share the assets you used for the app, i'm aware that it's not possible to change them without having to edit the app itself on your discord dev account

**Comment by ananagame** [comment]
Maybe later
